website/docs/user_guide/environments.md

```diff
@@ -56,6 +56,27 @@ In order for the SDK to download the feature toggle configuration for the correc
 
 ![An API key creation form. The form's fields are "username", "token type", "project", and, crucially, "environment". The development environment is selected.](/img/environments_api_keys.png 'Create Environment specific API Keys')
 
+## Cloning environments
+
+:::note Availability
+
+Environment cloning is an upcoming feature, and is scheduled to become available in one of the next few releases.
+
+:::
+
+Unleash environments can be cloned. Cloning an environment creates a **new** environment based on the selected source environment. When cloning an environment, you select any number of projects whose feature toggle configurations will also be cloned. These projects will have identical configurations for the source and target environments. However, the environments can then be configured independently and will not stay in sync with each other.
+
+When cloning an environment, you must give the new environment
+- a name
+- an environment type
+- a list of projects to clone configurations in
+
+You must also decide whether you want to clone the user permissions to the new environment or to make it a fresh environment where only an admin or editor of the selected projects can make changes. You can always add permissions later, using [Custom Project Roles](../user_guide/rbac#custom-project-roles).
+
+In order to clone an environment, you can follow the [How to clone environments](../how-to/how-to-clone-environments.mdx) guide.
+
+Once created, the new environment works just as any other environment.
```

Comment:
```suggestion
Once cloned, the new environment works just as any other environment.
```

Reply:
I don't agree with this suggestion. In my mind, the **old** environment is **cloned,** the **new** environment is **created**.

Reply:
I'm not sure I agree with this suggestion. I feel like the "cloned" environment is the source environment (the "clone" being the new one), and that when you clone an environment, you do create a new one regardless.